2025-01-08 13:49:13,285 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:49:13,653 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 13:49:13,798 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 13:49:26,259 - urllib3.util.retry - DEBUG - Incremented Retry for (url='/token'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:49:26,259 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)'))': /token
2025-01-08 13:49:26,259 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (2): oauth2.googleapis.com:443
2025-01-08 13:49:37,603 - urllib3.util.retry - DEBUG - Incremented Retry for (url='/token'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:49:37,603 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)'))': /token
2025-01-08 13:49:37,603 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (3): oauth2.googleapis.com:443
2025-01-08 13:50:47,443 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:50:47,583 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 13:50:47,599 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 13:53:13,265 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:53:13,405 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 13:53:13,421 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 13:53:15,077 - urllib3.connectionpool - DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/11" 200 None
2025-01-08 13:53:15,093 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): www.googleapis.com:443
2025-01-08 13:53:16,452 - urllib3.connectionpool - DEBUG - https://www.googleapis.com:443 "GET /drive/v3/files?q=mimeType%3D%22application%2Fvnd.google-apps.spreadsheet%22+and+name+%3D+%22gestion-reservas-dp%22&pageSize=1000&supportsAllDrives=True&includeItemsFromAllDrives=True&fields=kind%2CnextPageToken%2Cfiles%28id%2Cname%2CcreatedTime%2CmodifiedTime%29 HTTP/11" 200 None
2025-01-08 13:53:16,452 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): sheets.googleapis.com:443
2025-01-08 13:56:16,396 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:56:16,536 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 13:56:16,536 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 13:56:42,321 - urllib3.util.retry - DEBUG - Incremented Retry for (url='/token'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:56:42,321 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)'))': /token
2025-01-08 13:56:42,321 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (2): oauth2.googleapis.com:443
2025-01-08 13:57:34,384 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): ip-api.com:80
2025-01-08 13:57:36,748 - urllib3.connectionpool - DEBUG - http://ip-api.com:80 "GET /json/198.1.1.1 HTTP/11" 200 299
2025-01-08 13:58:54,733 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:58:54,889 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 13:58:54,905 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 13:59:00,472 - urllib3.connectionpool - DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/11" 200 None
2025-01-08 13:59:00,484 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): www.googleapis.com:443
2025-01-08 13:59:08,393 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 13:59:08,549 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 13:59:08,565 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 13:59:45,858 - urllib3.connectionpool - DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/11" 200 None
2025-01-08 13:59:45,873 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): www.googleapis.com:443
2025-01-08 14:01:25,885 - urllib3.util.retry - DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
2025-01-08 14:01:26,036 - google.auth.transport.requests - DEBUG - Making request: POST https://oauth2.googleapis.com/token
2025-01-08 14:01:26,052 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-01-08 14:02:09,469 - urllib3.connectionpool - DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/11" 200 None
2025-01-08 14:02:09,485 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): www.googleapis.com:443
